common/url.go

```diff
@@ -658,6 +662,32 @@ func mergeNormalParam(mergedUrl *URL, referenceUrl *URL, paramKeys []string) []f
 	return methodConfigMergeFcn
 }
 
+// doesn't encode url reserve character, url.QueryEscape will do this work
```

Comment:
I do now know what ur meaning from this explanation. maybe u'd better add some code examples.

Reply:
see next line “reference: https://github.com/golang/go.git, src/net/url/url.go, Encode method”